### DISCUSSION TITLE
From one developer to another thank you for your work and bravery

### DISCUSSION THREAD

**Discussion by ttytm** [JohnnyMorganz/StyLua#633] [Show and tell]
Besides creating a great formatter, you also have to have the courage to go against the stream and make the right choice and use tabs as the default, so hopefully the developer experience comes first more often and people are brought to their senses.  

I would even go so far as to say that the use of spaces is comparable to the use of Windows as an operating system. Something that has become the (in many cases crippling) standard as it is introduced as the default for most users (in terms of space indentation, it is still the default for most IDEs). With tab indentation, the developer's experience comes first, and I think that the people who know it just know it, and that there is no need to delve deeper into the matter, so there is nothing more to say than a thank you to brother.

<sub>(Of course, these are just opinions, and I'm exaggerating a bit here. But I still wanted to get rid of a thank you 🙏 .</sub>

**Comment by helgoboss** [comment]
I agree. Thank you so much, @JohnnyMorganz! StyLua is awesome. I don't care about spaces vs. tabs personally but the fact that this is written in Rust just made integration super easy for me as I have a project in which I generate Luau type definitions from Rust. Thanks again, also for luau-lsp!
